1908-01-20.xml

```diff
@@ -0,0 +1,1344 @@
+                      <p>There has just been placed on exhibition, in the mummy rooms at the British Museum, 
+                          the mummy of a lady, one of the famous priesthood of Amen Ra, at Karnak. The mummy, 
+                          which is undoubtedly one of the finest in the collection, is covered with a cartonnage 
+                          casing, modelled in the form of the body : the face, being a portrait of the deceased, 
+                          is thickly overlaid with gold, the eyes and eyebrows being inlaid with obsidian. The rest 
+                          of the casing is occupied with representations of the deceased addressing prayers to 
+                          Osiris Anubis and the genii of the dead, the whole being thickly crated with gold paint. 
+                          The coffin is, like most of its class, made in the shape of a mummified body, the sides 
+                          being ribly decorated with gilded representations of the gode The face on the lid is, 
+                          like that of the cartonnage, &amp; portrait of the deceased. The date of this interesting 
+                          addition to the museum is about B. C. 1,000.</p>
+                    </div>
+                    <div type="item">
+                          <head>A Complaint from Tantah.</head>
+                     <p> A correspondent writes from Tantsb to com plain of the disgraceful state of the Garayariyeh 
+                      Canal which runs through that town. A "Sadd" has been put in near the railway line to 
+                      Mansourah, as a bridge is being built, and this has cut off the running water. All through 
+                      the town the water is stagnant, quite black, and of a most unhealthy and objectionable 
+                      odour, and yet none of those who receive comparatively high salaries to attend to such 
+                      matters are ap. parently aware the facts-it they are, they are guilty of gross and 
+                      unpardonable breach of duty. To add to the natural stagnancy of the water the natives make 
+                      the canal a recipient for all household detritus, and to this disgraceful state of affairs,
+                      this negligence of public health, officials, high and low, the police, the tanzim, indeed 
+                      everyone, close their eyes. It is really time that our officials of position realised 
+                      that the day is past, when we shall overlook their Negligence with the too oft heard 
+                      "Malaseb it's Egypt."</p>
+                    </div>
+                      <div type="item">
+                      <head>PARIS-GENEVA HOUSE.</head>
+                      <p>Mm. Auberge is exhibiting her spring models at the Khedivial Hotel, Alexandria until 
+                      the 
+                      25th inst.
+                      
+                  </p> </div>
```

Comment:
add another `</div>` here to close the Local and General section--the next column is a fresh, independent item.